lib/class-wp-theme-json-resolver.php

```diff
@@ -309,6 +309,7 @@ public static function get_core_data() {
 	public static function get_theme_data( $theme_support_data = array() ) {
 		if ( null === self::$theme ) {
 			$theme_json_data = self::read_json_file( self::get_file_path_from_theme( 'experimental-theme.json' ) );
+			$theme_json_data = apply_filters( 'theme_json_data', $theme_json_data );
```

Comment:
I couldn't find other filters in this file so I wasn't sure if there was a reason we don't have them, or if there's a better alternative.